theme/static/js/hs-vue/left-header-app.js

```diff
@@ -123,6 +123,11 @@ Vue.component('add-author-modal', {
         }
     },
     methods: {
+        clearErrors: function(event){
+            if(event.target.className === "remove"){
```

Comment:
@Maurier I know this approach to clearing errors is a bit hacky.
Would be best to bind on the actual author input element. However, that element is generated using a ModelForm in the Vue template, so I'm not sure how to bind a listener on it. If you have a better solution, I would love to learn.

Reply:
We can probably improve this. Let's work on this together on a quick call soon. I'll message you to coordinate.